mappings/net/minecraft/client/render/entity/EntityRendererExplodingWitherSkull.mapping

```diff
@@ -1,3 +1,9 @@
 CLASS none/bvb net/minecraft/client/render/entity/EntityRendererExplodingWitherSkull
+	FIELD a INVINSABLE_SKIN Lnone/kp;
```

Comment:
`INVINCIBLE` (Did you check yet if that's the same name we're using for the wither skull property that causes them to destroy almost any block? Because that's what this is. Should stay consistent.)